### PR TITLE
ORLP-180

### PR DESCRIPTION
**Pull request by b0hdan**
Ability to uncheck decks in category and therefore to remove unchecked decks from user's cabinet has been added.

### CODE REVIEW

File: src/app/page/categoryInfo/deck/deck.component.html [hunk 1]
```diff
@@ -17,13 +17,14 @@
...
-        <div (click)="addDeckToFolder(deck.deckId)" *ngIf="!deck.status; else other_content">
-          <div *ngIf="isAuthorized()" class="button addDeckToUserButtonActive" title="Add deck to cabinet"></div>
+        <div *ngIf="!deck.status">
+          <div (click)="addDeckToFolder(deck.deckId)" *ngIf="isAuthorized()" class="button addDeckToUserButtonActive"
+               title="Add deck to cabinet"></div>
         </div>
-        <ng-template #other_content>
-          <div *ngIf="isAuthorized()" class="button addDeckToUserButtonChecked"
+        <div *ngIf="deck.status">
+          <div (click)="addDeckToFolder(deck.deckId)" *ngIf="isAuthorized()" class="button addDeckToUserButtonChecked"
```
Comment (zadorovskyi) [inline src/app/page/categoryInfo/deck/deck.component.html]: Is it possible to use here ng else statement?

File: src/app/page/categoryInfo/deck/deck.component.ts [hunk 2]
```diff
@@ -90,7 +90,7 @@ export class DeckComponent implements OnInit {
   changeDeckStatus(deckId: number) {
     for (const entry of this.decksWithStatus) {
       if (entry.deckId === deckId) {
-        entry.status = true;
+        entry.status = !entry.status;
```
Comment (antonbakalets) [inline src/app/page/categoryInfo/deck/deck.component.ts]: Optionally, I would add a break here to exit loop as soon as the required entry is found.

**Comment by koldovsky** [comment]
Please add task ID from Jira

File: src/app/page/categoryInfo/deck/deck.component.html (hunk 1 above)
Comment (b0hdan) [reply src/app/page/categoryInfo/deck/deck.component.html]: I don't think so. It was done with ng else statement but the error was shown, so I replaced it with just *ngIf statement.

File: src/app/page/categoryInfo/deck/deck.component.ts (hunk 2 above)
Comment (antonbakalets) [reply src/app/page/categoryInfo/deck/deck.component.ts]: You can commit to the same branch, the pull request should update automatically.

**Pull request merged**